test/TensorFlow/tensor_autodiff.swift

```diff
@@ -1,11 +1,12 @@
 // RUN: %target-swift-frontend -emit-sil %s | %FileCheck %s
-// XFAIL: *
 
 import TensorFlow
 
 public func test1() {
-  func addSelf(_ x: Tensor<Float>) -> Tensor<Float> {
-    return log(x)
+  func matsquare(_ x: Tensor<Float>) -> Tensor<Float> {
+    return matmul(x, x)
   }
-  _ = #gradient(addSelf)
+  _ = #gradient(matsquare)
```

Comment:
This is not being partitioned because tensor code is in an unused private function. Try to call it on some square matrix, and pass -O to the frontend.